packages/vite/src/node/cli.ts

```diff
@@ -28,6 +28,27 @@ interface GlobalCLIOptions {
   force?: boolean
 }
 
+const normalizeOptionsConfig: Record<string, (v: any[]) => any> = {
+  port: (v) => {
+    return v.find((p) => typeof p === 'number' || typeof p === 'string') || 5173
```

Comment:
Maybe `findLast` should be used here? Because if the other options are arrays, the last value you select.

Reply:
yep, but findLast cannot be supported in nodejs, so